### DISCUSSION TITLE
Background problem

### DISCUSSION THREAD

**Discussion by gipetkov** [marcokreeft87/room-card#238] [Q&A]
Hi, I have the problem with image background -only show in icon area : 

type: custom:room-card
title: Първи етаж
icon: mdi:home-floor-a
show_icon: true
styles:
  background-image: url('/local/11.jpg')
entity: switch.shelly_shsw_l_98cdac1e805d
rows:
  - entities:

![image](https://github.com/marcokreeft87/room-card/assets/77454148/1a745d04-ded5-415b-a1f9-3e67ceaec277)

**Comment by marcokreeft87** [comment]
Try 
```
card_styles
```

**Reply by gipetkov** [reply]
> Try
> 
> ```
> card_styles
> ```

Work. Thanks !
